best-practices.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Best Practices
+description: Learn about our best practices for building and maintaining your Chalk solution
+---
+
+# Best Practices
+
+With Chalk, the same solution can be implemented in a number of different ways. Below are some guidelines
+illustrating recommended patterns for building and maintaining your Chalk solution!
+
+## Data Sources/Integrations
+
+### Test your datasource connections
+
+Define integrations through the Chalk dashboard and check that they're connecting properly using the test datasource button.
+![Test Datasource](/img/test-datasource.png)
+
+### Avoid naming your datasources by their "type", e.g. don't call your postgres datasource "postgres"
+
+Though this works, it can lead to ambiguity in SQL resolvers when you have multiple data sources of the same type. This occurs because only one instance of a data source type, [Chalk lets you refer to that data source by it's type instead of its name](/fraud-3#online-data)
```

Comment:
missing words? 
 Chalk only lets you  refer to a data source by its type instead of its name when there is only one instance of a data source type? 
 
 (also its vs it's)